src/components/text/List/Item/Item.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { BodyText } from '../../BodyText/BodyText'
+
+export interface ItemProps {
+  /**
+   * Text to display in the Item. Can contain stylized text using `TextStyle`.
+   */
+  children: React.ReactNode
+}
+
+/**
+ * Item to display in a List.
+ */
+export const Item: React.FC<ItemProps> = ({ children }) => <BodyText>{children}</BodyText>
```

Comment:
I was expecting `Bullet` to be inside `Item`, rather than beside.
Maybe we could merge them? Or at least move `Item` inside `List.tsx` (since it's so simple)?

Reply:
The reason why `Bullet` is not inside `Item` is because `Item` does not control the bullet type. We don't want the user to do stuff like:
```tsx
<List>
  <List.Item type="number">A pizza</List.Item>
  <List.Item type="number">A smoothie</List.Item>
  <List.Item type="bullet">A tomato</List.Item>
<\List>
```

We could move the Bullet rendering in Item if we make the `List` a React Context, which could transparently provide props to children like the item type via a Hook. However, this will add "a tiny bit" of complexity.

Also, I prefer to leave `Item` in a separate file, as it is a User-facing component, which makes it easy to understand what is this component from the code. Also, it is simple right now but may grow in complexity over time.

Reply:
Let's keep it as is then 👍